### PR TITLE
Joop sessions

### PR DESCRIPTION
**Pull request by JoopStark**
## Description
started parties#new redirects and response need to be fixed
## Type of Change

- [x] **fix**
- [x] **feat**
- [ ] **test**
- [ ] **refactor**
- [ ] **docs**

## Checklist

- [x] **Code has been self reviewed**
- [ ] **Code runs without any errors**
- [ ] **Thorough testing has been implemented if adding feature**
- [ ] **All tests pass**

### CODE REVIEW

File: app/facades/party_facade.rb
Comment (reedhillmar) [inline app/facades/party_facade.rb:1]: We should compare FE/BE facades and make sure they match (ie our code 'looks similar', not that it's the same code)

File: app/views/parties/new.html.erb
Comment (reedhillmar) [inline app/views/parties/new.html.erb:1]: This is a big one, nicely done!

File: app/views/parties/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  Thriller </br>
+  <%= check_box_tag "genres []", 10752 %>
+  War </br>
+  <%= check_box_tag "genres []", 37 %>
+  Western </br>
+  
+  <%= f.label :max_duration %>
+  <%= f.number_field :max_duration, value: 300 %></br>
+
+  Rating:
```
Comment (reedhillmar) [inline app/views/parties/new.html.erb:60]: Do we want to deal with defaults? Are we storing a user's preferences right now? I know it was an idea but can't remember if we made it a stretch goal or MVP

File: spec/features/parties/new_spec.rb
Comment (reedhillmar) [inline spec/features/parties/new_spec.rb:1]: Let's tag team this today. BE will work on our side of it then get with you to make sure the testing works.

**Review by ethanb1145 — APPROVED** [review]
Nice job changing the party params in the create so that it follows a one to many.

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -79,3 +79,10 @@
   config.filter_sensitive_data('<PROPUBLICA_API_KEY>') { ENV['PROPUBLICA_API_KEY'] }
```
Comment (reedhillmar) [inline spec/rails_helper.rb:79]: Why is this line in here? FE shouldn't need to filter any data since it's not calling external API?

**Review by reedhillmar — APPROVED** [review]
I have some questions, but nothing that stops us from merging

**Pull request merged**